docs/topics/modules/changing_project_code.adoc

```diff
@@ -3,7 +3,6 @@
 
 The example project code includes a new route for *eventbus* and its supporting method. The project sets up *eventbus* to only allow outbound communications for the *my-feed* channel. Messages arriving in the *my-feed* channel are pushed to the browser using the SockJS bridge. This project includes a `vertx-eventbus.js` file along with the `index.html` file and also includes tweaks to the `index.html` file.
 
-
```

Comment:
@rkratky @Preeticp is this a format we're supposed to be using for all our procedures in modules? I keep meaning to ask since I didn't put these free headers here but someone did and I'm not sure what the reasoning is and so I don't do it but maybe I should be doing it?